README.md

```diff
@@ -1,116 +1,208 @@
-- A summary of collected statistics that might help one decide to continue using the tool is available in the `jfstmerge.summary` file.
+---
 
-#### Running with _git_
+## Conflict Handlers
 
-After installation, the tool is automatically integrated with git, with no need for further configuration. Then every time you invoke the `git merge` command, the tool is executed.
 
-#### Running standalone
+Is semistructured merge the ultimate solution? Not so easy. Despite reducing a lot of textual merge's false positives and false negatives, it introduces others on its own! We call them **extra** false positives/negatives. To help fixing those, S3M runs special mechanisms after every merge named as **Conflict Handlers**: algorithms that fix some semistructured merge potential flaws.
 
-Use the jar from the [/binary](https://github.com/guilhermejccavalcanti/jFSTMerge/tree/master/binary) folder, or from the installed folder.
+### [Deletions Handler](https://github.com/guilhermejccavalcanti/jFSTMerge/blob/master/src/main/java/br/ufpe/cin/mergers/handlers/DeletionsHandler.java)
 
-* Merging 3 files:
+* **Description** Every declaration removal by a contribution is problematic for semistructured merge as the correspondent node doesn't exist in the contribution's tree. This handler focuses on deletions of **inner classes**.
+* **Behavior**
+    If a contribution edited an inner class while another deleted it, we check if the latter also included a reference to the edited class. If true, we keep both the original and edited classes. Otherwise, we check if they're "sufficiently similar". If true, we join them to avoid duplications, otherwise, we report a conflict.
 
-  `java -jar pathto/jFSTMerge.jar -f "mine" "base" "theirs" -o "output"`
+### [Initialization Blocks Handler](https://github.com/guilhermejccavalcanti/jFSTMerge/blob/master/src/main/java/br/ufpe/cin/mergers/handlers/InitializationBlocksHandler.java)
+* **Description** Semistructured merge is unable to match **initialization blocks** (static and non-static), as it uses identifiers for this purpose and these elements have no identifiers, creating duplicates at the end. This handler tries to overcome this issue, matching nodes by content similarity.
+* **Behavior** If there's only one initialization block in all contributions, they're matched. Otherwise, for each initialization block in base, we search the most similar block in left and in right and merge them using textual merge.
 
-Where *mine*, *base*, *theirs* and *output* are filepaths.
-The attribute -o is optional, if omitted, *theirs* is used as the output file.
+### [Method and Constructor Renaming or Deletion Handler](https://github.com/guilhermejccavalcanti/jFSTMerge/blob/master/src/main/java/br/ufpe/cin/mergers/handlers/MethodAndConstructorRenamingAndDeletionHandler.java) 
```

Comment:
nao diz o problema. diz quando o handler é chamado e o que ele faz. 

"For each method or constructor renamed or deleted" como sabe se foi renomeado ou deletado? descreve o algoritmo mesmo para nao gerar confusao. para cada no em base mas nao em ...